layers/meta-resin-beaglebone/recipes-core/images/resin-image.inc

```diff
@@ -1,4 +1,4 @@
-IMAGE_FSTYPES_append_beaglebone = " resinos-img"
+IMAGE_FSTYPES_append_beaglebone = " resinos-img resin-sdcard.bmap"
```

Comment:
maybe change the name to something like "resinos-img.bmap" now that we don't use the resin-sdcard naming anymore?